src/styles/blocks/card.scss

```diff
@@ -0,0 +1,105 @@
+.card {
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```

Comment:
The better practice is to move this property to the * selector and remove it from all other places to avoid code duplication.

Reply:
Not resolved